examples/view_clearcontrol_dataset.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import tempfile
+import time
+
+from iohub.clearcontrol import (
```

Comment:
Importing a test fixture may be troublesome when we migrate these example scripts to the documentation page.